lighthouse-cli/test/smokehouse/test-definitions/pwa-svgomg.js

```diff
@@ -23,9 +23,14 @@ const config = {
  */
 const expectations = {
   lhr: {
-    requestedUrl: 'https://jakearchibald.github.io/svgomg/',
+    // Intentionally start out on http to test the redirect.
+    requestedUrl: 'http://jakearchibald.github.io/svgomg/',
```

Comment:
We have a few `redirects` smoke tests, should it go there instead?

Otherwise, please add an exclusion to these expectations so we can still run these smokes in devtools runner.